resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/features/BasicDriverFeatureStepSuite.scala

```diff
@@ -108,6 +108,11 @@ class BasicDriverFeatureStepSuite extends SparkFunSuite {
         envVar.getValueFrom.getFieldRef.getApiVersion.equals("v1") &&
         envVar.getValueFrom.getFieldRef.getFieldPath.equals("status.podIP")))
 
+    assert(configuredPod.container.getEnv.asScala.exists(envVar =>
+      envVar.getName.equals(ENV_DRIVER_POD_NAME) &&
```

Comment:
If we need this, this should be handled separately before this PR.

Reply:
removed